etna/transforms/timestamp/holiday.py

```diff
@@ -42,16 +74,17 @@ def __init__(self, iso_code: str = "RUS", mode: str = "binary", out_column: Opti
         iso_code:
             internationally recognised codes, designated to country for which we want to find the holidays
         mode:
-            `binary` to indicate holidays, `category` to specify which holiday do we have at each day
+            binary to indicate holidays, category to specify which holiday do we have at each day
         out_column:
-            name of added column. Use ``self.__repr__()`` if not given.
+            name of added column. Use `self.__repr__()` if not given.
         """
         super().__init__(required_features=["target"])
         self.iso_code = iso_code
         self.mode = mode
         self._mode = HolidayTransformMode(mode)
         self.holidays = holidays.country_holidays(iso_code)
         self.out_column = out_column
+        self.freq = ""
```

Comment:
I think, It's better to define `Node` as default value for `freq`.

Reply:
If I set self.freq = None it causes different warnings

Reply:
Can you give me example of warning?

Reply:
<img width="1792" alt="image" src="https://github.com/etna-team/etna/assets/122524141/5f3f9842-c78b-4638-a1c9-5d1ad6f677ff">

Reply:
It's better to share log of error, not screenshot

Reply:
Okey, let's stay `self.freq = ''`